tools/proteore_build_protein_interaction_maps/build_protein_interaction_maps.xml

```diff
@@ -0,0 +1,311 @@
+<tool id="build_protein_interaction_maps" name="Build Protein interaction network" version="2020.02.06">
+<description>[BioGRID, BioPlex, HuMAP]</description>
```

Comment:
maybe a text like `for BioGRID, BioPlex, HuMAP` is better than the square brackets?